autoPyTorch/data/tabular_feature_validator.py

```diff
@@ -14,67 +14,51 @@
 from sklearn.exceptions import NotFittedError
 from sklearn.impute import SimpleImputer
 from sklearn.pipeline import make_pipeline
-from sklearn.preprocessing import OneHotEncoder, StandardScaler
+from sklearn.preprocessing import OrdinalEncoder
 
 from autoPyTorch.data.base_feature_validator import BaseFeatureValidator, SUPPORTED_FEAT_TYPES
 
 
 def _create_column_transformer(
     preprocessors: Dict[str, List[BaseEstimator]],
-    numerical_columns: List[str],
     categorical_columns: List[str],
 ) -> ColumnTransformer:
     """
     Given a dictionary of preprocessors, this function
     creates a sklearn column transformer with appropriate
     columns associated with their preprocessors.
-
```

Comment:
I think it is better to have a space between description, Args, and Returns (because Google style follows it).
There are many where you removed space, so could you get them back?
It also makes easier to focus on the core changes in the GitHub GUI

Reply:
yeah I'll fix this, thanks for pointing out.